package.json

```
@@ -107,6 +107,7 @@
     "test": "npm-run-all build -p -r test:*",
```

I think this now needs to change to this:


```suggestion
    "test": "npm-run-all build test:unit -p -r test:e2e test:server",
```

---

This is because the `-p -r` runs them in parallel (`-p`) but also kills the other when the first finishes (`-r`) - which is basically used to kill the (otherwise never completing) server once e2e tests finish.

Without moving the `test:unit` to it's own section, it is the first to complete and kills the `e2e` and `server`, ending the test far too early.

---

Ah, good to know. [Updated](https://github.com/GoogleChrome/web-vitals/compare/86194e67c7f3745565f6057a488b154387a9c080..b099e8606a611b3d886736351cb60d27b440b2ce).